woodwork/accessor_utils.py

```diff
@@ -9,11 +9,11 @@
 
 def init_series(series, logical_type=None, semantic_tags=None,
                 use_standard_tags=True, description=None, metadata=None):
-    """Initializes Woodwork typing information for a Series, returning a new Series. The dtype
+    """Initializes Woodwork typing information for a series, numpy.ndarray or pd.api.extensions.ExtensionArray, returning a new Series. The dtype
```

Comment:
This line is a bit too long. A good guide is that you don't want to have to scroll to see the whole line if you have the screen split or something (an upper limit is probably 100 characters, but you'll see we're not perfect about sticking to that all the time).

You can split it up into multiple lines and just even out the rest of this section of the docstring.